README.md

```diff
@@ -1,6 +1,6 @@
 # Metal
 
-Prototype compiled language built in Rust with LLVM.
+A beautiful, robust, flexible, efficient general-purpose programming language with zero garbage collection built with LLVM and Rust.
```

Comment:
now that's just false, we don't have any of that

Reply:
Uh, idk what you mean?

beautiful: it doesn't exist, so the sky is beautiful
robust: you do anything with it, since it doesn't exist
flexible: use it anywhere! it doesn't exist, so it doesn't matter
efficient: it's the fastest programming language ever since, well, it doesn't exist
zero garbage collection: it doesn't exist, so it can't have garbage collection anyway

Reply:
i'd rather just revert the change